src/applications/ivc-champva/10-7959C/chapters/healthInsuranceInformation.js

```diff
@@ -46,211 +47,295 @@ schemas
-  const property = isPrimary ? 'hasPrimary' : 'hasSecondary';
   return {
     uiSchema: {
-      applicants: { items: {} },
-    },
-    schema: applicantListSchema([], {
+      ...titleUI(
+        ({ formData }) =>
+          `${nameWording(formData)} ${
+            isPrimary ? 'primary' : 'secondary'
+          } health insurance`,
+      ),
       [keyname]: {
-        type: 'object',
-        properties: {
-          [property]: { type: 'string' },
-          _unused: { type: 'string' },
-        },
+        ...yesNoUI({
+          updateUiSchema: formData => {
+            const useFirstPerson = formData.certifierRole === 'applicant';
+            return {
+              'ui:title': `${
+                useFirstPerson
+                  ? 'Do you'
+                  : `Does ${nameWording(formData, false)}`
+              } need to provide or update any other health insurance coverage?`,
+              'ui:options': {
+                hint: additionalFilesHint,
+              },
+            };
+          },
+        }),
       },
-    }),
+    },
+    schema: {
+      type: 'object',
+      required: [keyname],
+      properties: {
+        [keyname]: yesNoSchema,
+      },
+    },
   };
 }
 
 export function applicantProviderSchema(isPrimary) {
-  const keyname = isPrimary
+  const keyname1 = isPrimary
     ? 'applicantPrimaryProvider'
     : 'applicantSecondaryProvider';
-  return {
-    uiSchema: {
-      applicants: {
-        'ui:options': { viewField: ApplicantField },
-        items: {
-          ...titleUI(
-            ({ formData }) =>
-              `${applicantWording(formData)} ${
-                isPrimary ? '' : 'secondary'
-              } health insurance provider’s name`,
-          ),
-          [keyname]: {
-            'ui:title': 'Provider’s name',
-            'ui:webComponentField': VaTextInputField,
-          },
-        },
-      },
-    },
-    schema: applicantListSchema([keyname], {
-      titleSchema,
-      [keyname]: { type: 'string' },
-    }),
-  };
-}
-
-export function applicantInsuranceEffectiveDateSchema(isPrimary) {
-  const keyname = isPrimary
+  const keyname2 = isPrimary
     ? 'applicantPrimaryEffectiveDate'
     : 'applicantSecondaryEffectiveDate';
-  return {
-    uiSchema: {
-      applicants: {
-        'ui:options': { viewField: ApplicantField },
-        items: {
-          ...titleUI(
-            ({ formData }) =>
-              `${applicantWording(formData)} ${insuranceName(
-                formData,
-                isPrimary,
-              )} ${
-                isPrimary ? 'primary' : 'secondary'
-              } insurance effective date`,
-          ),
-          [keyname]: currentOrPastDateUI('Health insurance effective date'),
-        },
-      },
-    },
-    schema: applicantListSchema([keyname], {
-      titleSchema,
-      [keyname]: currentOrPastDateSchema,
-    }),
-  };
-}
-
-export function applicantInsuranceExpirationDateSchema(isPrimary) {
```

Comment:
Deleted this standalone effective date and expiration date screen configs because those dates are now displayed via the same config as `applicantProviderSchema`.